### PR TITLE
Extend VisionExcelConverter validation test to include nl mapping

### PR DESCRIPTION
**Pull request by bramstoeller**

### CODE REVIEW

File: src/power_grid_model_io/config/excel/vision_en.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
```
Comment (TonyXiang8787) [inline src/power_grid_model_io/config/excel/vision_en.yaml:1]: You don't need to add IO. The project is still Power Grid Model. `power-grid-model-io` is just one of the repositories and libraries.

**Comment by TonyXiang8787** [comment]
@bramstoeller looks good. PLease fix the error in the pipeline

**Review by jaapschoutenalliander — APPROVED** [review]
Tested with three different of our user-generated files and these all worked perfectly

**Comment by TonyXiang8787** [comment]
@jaapschoutenalliander can you try again. We have changed the cross-reference method in the backend. See https://power-grid-model-io.readthedocs.io/en/stable/converters/tabular_converter.html#autoid

**Review by TonyXiang8787 — APPROVED** [review]

**Pull request merged**